services/director-v2/tests/conftest.py

```diff
@@ -92,12 +93,50 @@ def mocks_dir(tests_dir: Path) -> Path:
 
 
 @pytest.fixture(scope="session")
-def sleepers_workbench_file(mocks_dir: Path) -> Path:
-    file_path = mocks_dir / "4sleepers_workbench.json"
+def fake_workbench_file(mocks_dir: Path) -> Path:
+    file_path = mocks_dir / "fake_workbench.json"
     assert file_path.exists()
     return file_path
 
 
 @pytest.fixture(scope="session")
-def sleepers_workbench(sleepers_workbench_file: Path) -> Dict:
-    return json.loads(sleepers_workbench_file.read_text())
+def fake_workbench(fake_workbench_file: Path) -> Workbench:
+    workbench_dict = json.loads(fake_workbench_file.read_text())
+    workbench = {}
+    for node_id, node_data in workbench_dict.items():
+        workbench[node_id] = Node.parse_obj(node_data)
+    return workbench
+
+
+@pytest.fixture(scope="session")
+def fake_workbench_as_dict(fake_workbench_file: Path) -> Dict[str, Any]:
+    workbench_dict = json.loads(fake_workbench_file.read_text())
+    return workbench_dict
+
+
+@pytest.fixture(scope="session")
+def fake_workbench_computational_adjacency_file(mocks_dir: Path) -> Path:
+    file_path = mocks_dir / "fake_workbench_computational_adjacency_list.json"
+    assert file_path.exists()
+    return file_path
+
+
+@pytest.fixture(scope="session")
+def fake_workbench_adjacency(
+    fake_workbench_computational_adjacency_file: Path,
+) -> Dict[str, Any]:
+    return json.loads(fake_workbench_computational_adjacency_file.read_text())
+
+
+@pytest.fixture(scope="session")
+def fake_workbench_complete_adjacency_file(mocks_dir: Path) -> Path:
+    file_path = mocks_dir / "fake_workbench_complete_adj_list.json"
```

Comment:
MINOR: when we are not going to use the files as fixtures or inputs to more than one, I guess it does not harm to just put it all in a single fixture , i.e. the file and the dic

Reply:
well I prefer having well separated concerns.